src/main/java/com/mineinabyss/mobzy/listener/MobListener.kt

```diff
@@ -137,9 +138,10 @@ object MobListener : Listener {
                 entity.get<Model>()?.apply { entity.equipment?.helmet = modelItemStack }
                 entity.removeScoreboardTag("customMob2")
                 entity.addScoreboardTag("customMob3")
-            } else if (entity.isCustomEntity && entity.toNMS() !is NPC && !entity.isCustomAndRenamed) {
+            } else if (entity.isCustomMob && entity.toNMS() !is NPC && !entity.isCustomAndRenamed) {
```

Comment:
Yeah this entire thing for migrations needs to be rewritten to not be awful. The thing to remove when far away is something I added a while back because server kept spawning in entities like crazy. It should be safer to remove now but I should do proper testing first.